lib/data_models/genre.dart

```diff
@@ -0,0 +1,17 @@
+class Genre{
+
+  final int id;
+  final String name;
+
+  Genre({
+  required this.id,
+  required this.name
```

Comment:
Trailing comma, and add a space between `Genre{` => `Genre {`

Reply:
The space is eliminated when I format my code with Ctrl + Alt + L. Should i fix it anyways and not format it after?